### DISCUSSION TITLE
Is this library ready for production?

### DISCUSSION THREAD

**Discussion by lbineau** [Accudio/async-alpine#43] [General]
I've quickly tested this library and I'll use it when I start a new project on a server side heavy oriented project. It really looks similar to Astro loading strategy that I love :-).
Do you consider it as ready for production? If yes it should definitly be an AlpineJS recommended plugins!

**Comment by Accudio** [comment]
It's absolutely ready for production!

It's being used in multiple sites on production, some examples here: [async-alpine.dev/showcase](https://async-alpine.dev/showcase/). [Filament PHP](https://github.com/filamentphp/filament) also includes Async Alpine and is used on a huge number of production sites.

Whilst not technically an AlpineJS Plugin at the moment — it's a wrapper rather than a plugin — I am working on a rework that would move it into proper plugin territory!

It's interesting you say it's similar to Astro, that was a major influence and during the initial alpha I chatted a little with Astro devs about the implementation and design! ;-)

**Comment by c-nv-s** [comment]
is it possible to locally build the documentation site? for people who don't have  constant internet access.

**Reply by Accudio** [reply]
@c-nv-s Good question, currently the documentation site isn't public. Not for any particular reason other than it being a bit rough and needing tidying up. When I can I'll work on getting that ready to be made public.

In the meantime there aren't many pages and you'll be able to easily manage it with an archiving tool!

**Reply by Accudio** [reply]
@c-nv-s I've made the repo for the documentation site public here: https://github.com/Accudio/async-alpine-website

**Reply by c-nv-s** [reply]
Thanks a lot
